src/_test_/index_test.js

```diff
@@ -152,11 +198,22 @@ describe('./index.js', function() {
             });
         });
 
+        describe('#getTaggedConfig', function() {
+            it('returns tagged config used in constructor', function() {
```

Comment:
I'd say this needs tests for the base case too .

Reply:
This simple getter is being tested under the ```CerebroConfig``` description block which just checks that it gets the right object and ignores evaluation, which is tested in the ```resolveConfig``` block with a true/false tagEvaluator.

Reply:
Ah cool cool.